build.sbt

```diff
@@ -1,7 +1,7 @@
 import Dependencies._
 
 ThisBuild / scalaVersion := "2.13.14"
-ThisBuild / version := "0.3.0-SNAPSHOT"
+ThisBuild / version := "1.0.0-SNAPSHOT"
```

Comment:
The new format breaks compatibility, so we increment the major component of the semantic versioning.